Source/Core/VideoCommon/GraphicsModSystem/Runtime/CustomPipeline.cpp

```diff
@@ -336,17 +336,20 @@ void CustomPipeline::UpdatePixelData(
               first_slice.m_levels[0].format, 0, texture_type);
           texture_asset->m_texture = g_gfx->CreateTexture(
               texture_config, fmt::format("Custom shader texture '{}'", property.m_code_name));
-          for (std::size_t slice_index = 0; slice_index < texture_data->m_texture.m_slices.size();
-               slice_index++)
+          if (texture_asset->m_texture)
```

Comment:
Is the `g_gfx->CreateTexture` above failable?

Reply:
I saw it fail when I ran out of memory.  In normal cases probably not.  And honestly, if you're out of memory...there are bigger issues (I will soon have a texture cache PR that reworks this to resolve that issue)